scopes/ui-foundation/cli/ink/standalone-new-line/index.ts

```diff
@@ -0,0 +1 @@
+export * from './standalone-new-line';
```

Comment:
Please remove this component. It doesn't do anything useful. you can just use the NewLine tag in your code.

Reply:
This is where it is useful:
<Box>
    ...
<Box/>
<NewLine/>
<Box>
    ...
<Box/>

**If it wasn't useful I wouldn't do it in the first place, right?**

Reply:
My point is that you wrap the built-in NewLine in <Text/> tags, that's all. It's a bit stretch IMO to create a dedicated component for this. Anyway, I searched in the code and you don't use it anyway, so I guess it can be removed.